builder/build.js

```diff
@@ -5,7 +5,7 @@ const config = require('./config');
 const screenshots = require('./plugins/screenshots');
 
 const server = serve({
-	port: 8082,
```

Comment:
Why though? Is it why CI fails?

Reply:
8083 allows your to run build and serve (on 8082) simultaneously

Reply:
Lol, ok, I just liked the part where screenshot grabber and server were on the same port LOL